.stylelintrc

```diff
@@ -0,0 +1,4 @@
+{
+  "_developer-comment": "this file to stay empty to ensure CI builds pass. editable .stylelintrc found at root/.stylelintrc",
+  "rules": {}
+}
```

Comment:
kinda weird but the build command failed without this, hence CI builds failed without it. open for other suggestions of how to accomplish this